### ISSUE TITLE
part of descriptions are missing

### ISSUE THREAD

**Issue by pank** [xbmc-danish-addons/plugin.video.drnu#74]
Hi

It seems that for long program descriptions, not the whole description is gotten.

E.g. for "Mørkelagt: Spionchefens hemmeligheder - del 1" the description ends with "men pludselig bliver han og en række andre". It should be

> Men pludselig bliver han og en række andre ledende medarbejdere i efterretningstjenesterne sendt hjem, blandt andet fordi Forsvarets Efterretningstjeneste bliver mistænkt for at have udspioneret danske statsborgere. 

https://www.dr.dk/drtv/episode/moerklagt_-spionchefens-hemmeligheder_345764

**Comment by TermeHansen** [comment]
you are right, for some reason they only expose shortDescription when getting the list of all the informarion we see now. To get full description we need to query each item - so a bit slower ui from this...

https://github.com/xbmc-danish-addons/plugin.video.drnu/pull/75

How does that look and feel?
